notebooks/04_simple_machine_learning_tangent_spaces.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<p>Actually, FrechetMean should be covered in tutorial 03. Maybe let's say: covered in the previous tutorial?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/816/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>